client/my-sites/site-settings/podcast-cover-image-setting/index.jsx

```diff
@@ -96,7 +96,7 @@ class PodcastCoverImageSetting extends PureComponent {
 				fileContents: blob,
 				fileName,
 			};
-			const uploadedMedia = await this.props.addMedia( file, this.props.site );
+			const [ uploadedMedia ] = await this.props.addMedia( file, this.props.site );
```

Comment:
`addMedia` returns an array so we have to destructure at this point to make sure we're calling `onSelect` with appropriate data